### DISCUSSION TITLE
Toniko Panjota using ToonSquid in IG Story

### DISCUSSION THREAD

**Discussion by otaworks** [keiwando/toonsquid#711] [Show and tell]
I saw this on Toniko's instagram story and wanted to share it. The link is below:
https://www.instagram.com/stories/tonikopantoja/3524539485185310344?utm_source=ig_story_item_share&igsh=emQ3dnQ0ZW93b3Rq

**Comment by otaworks** [comment]
https://github.com/user-attachments/assets/c8a1317b-ff62-44f8-9f50-d8a9bc5adb08

**Comment by budaloops** [comment]
Neat
